scripts/profiler/plot_nordic.py

```diff
@@ -448,15 +448,6 @@ def animate_events_real_time(self, fig, selected_events_types, one_line):
                 self.processed_events.tracked_events.append(TrackedEvent(event, None, None))
             else:
                 self.temp_events.append(event)
-                if event.timestamp > time.time() - self.start_time + self.draw_state.added_time - \
```

Comment:
Why dropping aligning?
If I recall correctly in some cases the plot was moving too slow and then aligning could be used to display the latest events